Examples/MAX32662/Demo/main.c

```diff
@@ -33,7 +33,7 @@
 
 /**
  * @file    main.c
- * @brief   Demo
+ * @brief   This example demonstrates the use of the TFT Display, UART Terminal, RTC, Pushbuttons, and LEDs on the MAX32662EVKIT.
```

Comment:
MAX32662EVKIT gets added in the common examples.
![image](https://github.com/Analog-Devices-MSDK/msdk/assets/75494566/16e1a626-42d3-46a7-987e-81879779e10a)

Reply:
Fixed https://github.com/Analog-Devices-MSDK/msdk/pull/683/commits/83fdfed973e788133e73a8ba476ac13e159964f7